autoPyTorch/api/base_task.py

```diff
@@ -443,14 +447,14 @@ def ensemble_performance_history(self) -> List[Dict[str, Any]]:
     def trajectory(self) -> Optional[List]:
         return self._results_manager.trajectory
 
-    def set_pipeline_config(self, **pipeline_config_kwargs: Any) -> None:
+    def set_pipeline_options(self, **pipeline_options_kwargs: Any) -> None:
```

Comment:
Any specific reasons to rename `configs` to `options`?

Reply:
It was confusing because at some places like [here](https://github.com/automl/Auto-PyTorch/blob/master/autoPyTorch/configs/default_pipeline_options.json), [here](https://github.com/automl/Auto-PyTorch/blob/c138dff1909464110f4d35dfb9c016e724a8c25a/autoPyTorch/api/base_task.py#L219), etc it is `options` but otherwise its `configs`. I wanted to unify the usage and I guess pipeline options made more sense than pipeline_config as config can be confused with the hyperparameter configuration. 

Feel free to suggest a new name.